src/utils/bytes.js

```diff
@@ -32,3 +32,28 @@ export function stringToBytes(str) {
   }
   return bytes;
 };
+
+/**
+ * Converts a 8-bit bytes array into a string
+ * @param {!Uint8Array} bytes
+ * @return {string}
+ */
+export function bytesToString(bytes) {
+  return String.fromCharCode.apply(String, bytes);
+};
+
+/**
+ * Generate a random bytes array with specific length
+ * @param {!number} length
+ * @return {!Uint8Array}
+ */
+export function getRandomBytesArray(length) {
```

Comment:
After discussion with @jridgewell and @lannka, we agreed to use the win.crypto.getRandomValues (which has enough entropy) by default if available, otherwise, fall back to use Math.random once only in the share tracking code(`Math.random` has limited precision and it's enough for random share tracking generation). The corresponding PR is #4678